src/state/configs.ts

```diff
@@ -132,6 +137,10 @@ export const configsSlice = createSlice({
         })
       );
     },
+    setShouldHideLaunchableMarkets: (state: ConfigsState, action: PayloadAction<boolean>) => {
```

Comment:
this is fine but why not use the new localstorage thing tina added?

Reply:
redux-persist? I think Aleka is adding in her Favorites PR

Reply:
I made sure to add this to the config slice that will be added to persist later

Reply:
sounds good